canonical-web-and-design/greenhouse-exporter
# Show workload in hours

## Done
Modify `api/workload` and the corresponding frontend table so it only shows the total workload for each person
## QA
- VPN
- `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`
- Go to `http://localhost:8501/`
- Check that the workload table 1) has data 2) there are two columns: interviewer and hours
 ## Issue
#49
